common/cmake/gen.cmake

```diff
@@ -116,15 +116,16 @@ function(MUX_GEN)
     DEPENDS
       ${PYTHON3} ${PYTHON3_TARGET}
       ${symbiflow-arch-defs_SOURCE_DIR}/utils/mux_gen.py
-      ${symbiflow-arch-defs_SOURCE_DIR}/vpr/muxes/logic/mux${MUX_GEN_WIDTH}/mux${MUX_GEN_WIDTH}.sim.v
+      #${symbiflow-arch-defs_SOURCE_DIR}/vpr/muxes/logic/mux${MUX_GEN_WIDTH}/mux${MUX_GEN_WIDTH}.sim.v
     COMMAND ${PYTHON3} ${symbiflow-arch-defs_SOURCE_DIR}/utils/mux_gen.py ${MUX_GEN_ARGS}
   )
 
   add_file_target(FILE "${MUX_GEN_NAME}.sim.v" GENERATED)
   add_file_target(FILE "${MUX_GEN_NAME}.pb_type.xml" GENERATED)
   add_file_target(FILE "${MUX_GEN_NAME}.model.xml" GENERATED)
 
-  add_custom_target(${MUX_GEN_NAME} DEPENDS ${OUTPUTS})
+  get_rel_target(REL_MUX_GEN_NAME muxgen ${MUX_GEN_NAME})
```

Comment:
Why is this not add_file_target?  I don't think you understand the implication of what you are doing here?

Reply:
This is the target name which depends on the other tile targets above. It generates a target of the name like `muxgen_utils_vlog_tests_muxes_routing_rmux`.

Reply:
The parameter NAME is now ambiguous.  Please rename NAME to something that implies that the target will not follow the name provided.  In this case TOP_MODULE might be a good parameter name, because it does specific the TOP_MODULE in the output verilog.

Reply:
@kgugala